src/Core/Batch/BatchTrait.php

```diff
@@ -200,4 +214,43 @@ private function setCommonBatchProperties(array $options)
             $this->batchOptions
         );
     }
+
+    /**
+     * @param array $options
+     * @return array
+     */
+    private function setWrappedClientConfig(array $options)
```

Comment:
Maybe this should be renamed to `getWrappedClientConfig()`? It's not setting anything.